enterprise/backend/src/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions.clj

```diff
@@ -264,45 +263,33 @@
   "Replace `:source-table` entries that refer to Tables for which we have applicable GTAPs with `:source-query` entries
   from their GTAPs."
   [m table-id->gtap]
-  ;; replace maps that have `:source-table` key and a matching entry in `table-id->gtap`, but do not have `:gtap?` key
+  ;; replace maps that have `:source-table` key and a matching entry in `table-id->gtap`, but do not have `::gtap?` key
   (mbql.u/replace m
-    (_ :guard (every-pred map? (complement :gtap?) :source-table #(get table-id->gtap (:source-table %))))
+    (_ :guard (every-pred map? (complement ::gtap?) :source-table #(get table-id->gtap (:source-table %))))
```

Comment:
does the auto resolved keyword `::gtap?` vs `:gtap?` fix anything or just a bit more descriptive in tests?

Reply:
it's just more descriptive. Doesn't change anything